### PR TITLE
Create topics/jest/index.md

### PR DESCRIPTION
**Pull request by WalterGR**
Jest is a JavaScript testing framework built on top of Jasmine, with a focus on simplicity and support for large web applications.

### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Explanation of proposed changes: Because the topic page shows "Here are 11,735 public repositories matching this topic..." (i.e. matching `jest`) yet it has no description.  (I am not interested in ongoing curation.)

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @WalterGR!

File: topics/jest/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: jest
```
Comment (MikeMcQuaid) [inline topics/jest/index.md:2]:
```suggestion
```

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
oops sorry just noticed a wee CI failure: https://github.com/github/explore/runs/6758378961?check_suite_focus=true#step:4:16

**Review by Drewski813 — COMMENTED** [review]
What up bro

File: topics/jest/index.md (hunk 1 above)
Comment (Athayde3586) [reply topics/jest/index.md:2]: I told you

**Comment by Athayde3586** [comment]
What's up

**Comment by Athayde3586** [comment]
U got something for me

**Review by MikeMcQuaid — APPROVED** [review]
Thanks @WalterGR!

**Pull request merged**